pkgs/development/libraries/nanopb/default.nix

```diff
@@ -7,53 +7,37 @@
 , stdenv
 }:
 
-let
-  pythonRuntime = python3.withPackages(ps: [ ps.protobuf ]);
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "nanopb";
-  version = "0.4.1";
+  version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = version;
-    sha256 = "16zxk42wzn519bpxf4578qn97k0h1cnbkvqqkqvka9sl0n3lz2dp";
+    sha256 = "sha256-32YU6LfPojvEVA2Z3DRZdGzDkSpAUgo1BueC/pl9qlY=";
   };
 
-  nativeBuildInputs = [ cmake python3 ];
+  nativeBuildInputs = [ cmake python3 python3.pkgs.wrapPython ];
+
+  pythonPath = with python3.pkgs; [ python3.pkgs.protobuf six ];
```

Comment:
```suggestion
  pythonPath = with python3.pkgs; [ protobuf six ];
```

Reply:
That doesn't work. `with` doesn't seem to override the protobuf from the outer scope.

Reply:
Good catch. I did not check this.